setup.py

```diff
@@ -351,6 +356,8 @@ def add_dir_name(dir_name, filenames):
     filename = swig_path.lib_filename
     if platform.system() == 'Linux':
```

Comment:
Seems that we use both os.name and platform.system() to determine the OS.
Which one should be actually used ? It's probably best to use a single one to have consistent results ?